jaxrpc-ri/impl/src/main/java/com/sun/xml/rpc/processor/modeler/j2ee/xml/Factory.java

```diff
@@ -363,7 +368,14 @@ public SimpleType createRootDOMFromSimpleType(
     private Object createRootDOMHelper(
         String classname,
         String rootElementName) {
-        document = new DocumentImpl();
+        try {
+            DocumentBuilderFactory dbf = DocumentBuilderFactory.newInstance();
+            document = dbf.newDocumentBuilder().newDocument();
+        } catch (ParserConfigurationException e) {
+            System.out.println("Exception: Factory::createRootDOMHelper() " + e);
```

Comment:
Do you want System.out.println here or logging?

Reply:
It's intentional to remain consistent with the rest of the code. I don't like it but I also do not want to spend time on updating the whole project to use java util logging (or alternative)